admin/services/queue-name-service.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+const config = require('../config')
+
+const NAMES = {
+  PREPARE_CHECK: 'prepare-check',
+  CHECK_COMPLETE: 'check-complete',
+  PUPIL_FEEDBACK: 'pupil-feedback'
+}
+
+Object.freeze(NAMES)
```

Comment:
why freezing it? is there a chance this can be mutated somehow?

Reply:
It's useful indeed, I am wondering what would be the scenario where this would be somehow modified.

Reply:
It's exported.   This prevents meddling.